docs/index.md

```diff
@@ -1,49 +1,40 @@
-[`solnlib`](https://github.com/splunk/addonfactory-solutions-library-python) and
-[`splunktaucclib`](https://github.com/splunk/addonfactory-ucc-library). More
-information [here](ucc_related_libraries.md).
+## Libraries
 
-## Features
+UCC-based add-ons are powered by the following Splunk libraries:
+* `solnlib`, see [https://github.com/splunk/addonfactory-solutions-library-python](https://github.com/splunk/addonfactory-solutions-library-python)
+* `splunktaucclib`, see [https://github.com/splunk/addonfactory-ucc-library](https://github.com/splunk/addonfactory-ucc-library). 
 
-The `splunk-add-on-ucc-framework`:
+For more information, see [UCC-related libraries](ucc_related_libraries.md).
 
-* generates UI (`appserver` folder).
-* generates Python REST handlers to support UI CRUD operations (`bin` folder).
-* generates [inputs](./inputs/index.md) and their [helper modules](./inputs/helper.md)
-* generates OpenAPI description documents (`appserver/static/openapi.json` file) (for more information, see [here](openapi.md)).
-* generates `.conf` files (more information, see [here](dot_conf_files.md)).
-* installs Python requirements (`lib` folder).
-* generate metadata files (`metadata` folder).
-* generates the monitoring dashboard (for more information, see [here](dashboard.md)).
-* it possibly extends the UI with custom codes (for more information, see [here](custom_ui_extensions/custom_hook.md)).
-* it possibly extends the build process via a `additional_packaging.py` file (more information, [here](additional_packaging.md)).
-* generates the necessary files defined for the Alert Action, if defined in globalConfig (for more informaiton, see [here](alert_actions/index.md)).
+## What UCC generates 
 
-## Installation
+When you use UCC to create an add-on, the following elements are generated and stored in the appropriate folders: 
 
-`splunk-add-on-ucc-framework` is available on [PyPI](https://pypi.org/project/splunk-add-on-ucc-framework/).
+* UI is stored in the `appserver` folder,
+* Python REST handlers that support UI CRUD operations are stored in the `bin` folder,
+* inputs and their helper modules. For more information, see [Inputs](./inputs/index.md) and [Helper modules](./inputs/helper.md),
+* OpenAPI description documents are stored in the `appserver/static/openapi.json` file. For more information, see [OpenAPI description document](openapi.md),
+* `.conf` files. For more information, see [.conf files](dot_conf_files.md),
+* Python requirements are installed in the `lib` folder,
+* metadata files are stored in the `metadata` folder,
+* the monitoring dashboard. For more information, see [Dashboard](dashboard.md),
+* the necessary files defined for the alert action <!---who defines the files? can I rewrite to: the files defined for the alert action? --->, if the alert action is defined in globalConfig <!--- is this a file name? --->. For more information, see [Alert actions](alert_actions/index.md). 
```

Comment:
We define the files that are generated (it's the main responsibility of the UCC framework) and developers define the configuration in the `globalConfig` file.